log/loki.go

```diff
@@ -282,7 +289,7 @@ func (h *lokiHook) loop() {
 		case <-h.ctx.Done():
 			ch := make(chan int64)
 			pushCh <- ch
-			ch <- 0
+			ch <- time.Now().Add(time.Second).UnixNano()
```

Comment:
Since this is the cutoff time, is the second to deal with time sync issues and get some last events? Should it be equal to the push period, so maybe `h.pushPeriod` here instead?

Reply:
yeah ... the basic idea is that ... as far as I can see you can call time.Now() in two parallel goroutines ... and you won't exactly get ... sequential times even if in reality they were ... so I added some time to get "hopefully" everything. 

I don't think that `h.pushPeriod` is better ... if anything I am more for `time.Hour` and a comment explaining why it is done at all (this took me 1 hour to figure out why I wasn't getting the last messages even though I was waiting for them and around 10 fmt.Printlns)